locust/readme.md

```diff
@@ -0,0 +1,113 @@
+
+# Locust tests README
+
+## Overview
+This README explains how to run Locust tests for `find-moj-data` and `datahub`
+
+## Prerequisites
+- Python 3.6+
+- Locust (install using `pip install locust`)
+
+## Project Structure
+```
+your_project_directory/
+│
+├── locustfile_datahub.py
+├── locustfile_find_moj_data.py
+├── README.md
+└── requirements.txt
+```
+
+## Installation
+1. **Clone the repository** (if you haven't already):
+    ```sh
+    git clone https://github.com/ministryofjustice/data-catalogue
+    cd data-catalogue
+    ```
+
+2. **Install the dependencies**:
+    ```sh
+    pip install -r requirements.txt
```

Comment:
I'm not too bothered right now, but we may want to swap this to poetry at some point, for consistency with the rest of the project

Reply:
good point. We should do it